plutus-tx/testlib/PlutusTx/Test.hs

```diff
@@ -140,28 +139,7 @@ renderExcess tData mData diff =
 -- Budget testing
 
 goldenBudget :: TestName -> CompiledCode a -> TestNested
-goldenBudget name compiledCode = do
-  path <- Reader.ask
-  let budgetText = measureBudget compiledCode
-      filename = foldr (</>) (name ++ ".budget.golden") path
-  pure $
-    goldenVsDoc name filename (maybe "Failed" pretty budgetText)
-
--- TODO: expose something like this somewhere more useful
-measureBudget :: CompiledCode a -> Maybe PLC.ExBudget
-measureBudget compiledCode =
-  let programE =
-        PLC.runQuote $
-          runExceptT @PLC.FreeVariableError $
-            traverseOf UPLC.progTerm UPLC.unDeBruijnTerm $
-              getPlcNoAnn compiledCode
-   in case programE of
-        Left _ -> Nothing
-        Right program ->
-          let (_, UPLC.TallyingSt _ budget) =
-                UPLC.runCekNoEmit PLC.defaultCekParameters UPLC.tallying $
-                  program ^. UPLC.progTerm
-           in Just budget
+goldenBudget name compiledCode = goldenUplcBudget name (getPlcNoAnn compiledCode)
```

Comment:
Does this compile? `getPlcNoAnn` returns a `Program` but `goldenUplcBudget` expects a `Term`.

Reply:
it did locally...

Reply:
oops sorry that's the `goldenBudget`, that's why. My HLS is not working well again today. Fixing.